index.js

```diff
@@ -22,7 +27,9 @@ module.exports = {
 
   init: function() {
     var bowerDeps = this.project.bowerDependencies();
+
     var VersionChecker = require('ember-cli-version-checker');
+    var options = this.options = this.options || {};
```

Comment:
Locally I change this line to use `{ babel: {} }` as the default option and tested out the resulting build. The built assets looked good.